spec/timeago_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'timeago'
+describe TimeAgo do
+  let(:under_minute) {Time.new - 59}
+  let(:under_hour) {Time.new - 3599}
+  let(:under_day) {Time.new - 86399}
+  let(:over_day) {Time.new - 86401}
```

Comment:
Separate every 3 digits in the integer portion of a number with underscores(_).<br>Space missing inside {.<br>Space missing inside }.<br>Do not use `Time.new` without zone. Use one of `Time.zone.now`, `Time.new.current`, `Time.new.in_time_zone`, `Time.new.utc`, `Time.new.getlocal`, `Time.new.iso8601`, `Time.new.jisx0301`, `Time.new.rfc3339`, `Time.new.to_i`, `Time.new.to_f` instead.